code/modules/admin/topic.dm

```diff
@@ -876,17 +876,21 @@
 			return
 
 		if (SSticker.HasRoundStarted())
-			if (askuser(usr, "The game has already started. Would you like to save this as the default mode effective next round?", "Save mode", "Yes", "Cancel", Timeout = null) == 1)
-				SSticker.save_mode(href_list["c_mode2"])
+			if(check_rights(R_DBRANKS, FALSE))
+				if(askuser(usr, "The game has already started. Would you like to save this as the default mode effective next round?", "Save mode", "Yes", "Cancel", Timeout = null) == TRUE)
```

Comment:
Does this not return the option they pressed, IE "Yes" or "Cancel"?

Reply:
I have no idea why but it returns 1 for yes and 2 for cancel